ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/alter_table.md

```diff
@@ -220,6 +220,18 @@ For example, this command disables automatic partitioning of the table:
 ALTER TABLE series SET (AUTO_PARTITIONING_BY_SIZE = DISABLED);
 ```
 
+Using the `ALTER` command, you can also modify the `TTL` parameters of the table. For example, to increase the delay interval for record deletion:
+```sql
+ALTER TABLE table_name 
+SET (TTL = Interval("PT10S") ON column_name);
```

Comment:
This example doesn't make sense as TTL works asynchronously and won't clean up data that fast. Usually, it's in like days, not seconds.